manifests/init.pp

```diff
@@ -38,7 +38,7 @@
   $purge_vhost_dir                                               = undef,
   $purge_vdir                                                    = false,
   $serveradmin                                                   = 'root@localhost',
-  $sendfile                                                      = 'On',
+  Enum['On', 'Off', 'on', 'off'] $sendfile                       = 'On',
```

Comment:
For simplification you could add a type ```Apache::OnOff``` since this validation is used in a lot of places.